cmd/kops/export_kubecfg.go

```diff
@@ -93,10 +97,22 @@ func RunExportKubecfg(f *util.Factory, out io.Writer, options *ExportKubecfgOpti
 		return err
 	}
 
-	conf, err := kubeconfig.BuildKubecfg(cluster, keyStore, secretStore, &commands.CloudDiscoveryStatusStore{})
+	conf, err := kubeconfig.BuildKubecfg(cluster, keyStore, secretStore, &commands.CloudDiscoveryStatusStore{}, buildPathOptions(options))
 	if err != nil {
 		return err
 	}
 
 	return conf.WriteKubecfg()
 }
+
+func buildPathOptions(options *ExportKubecfgOptions) *clientcmd.PathOptions {
+	pathOptions := clientcmd.NewDefaultPathOptions()
+
+	if len(options.KubeConfigPath) > 0 {
+		pathOptions.GlobalFile = options.KubeConfigPath
```

Comment:
I think maybe we should use `ExplicitPath` in `ClientConfigLoadingRules` ?  What does kubectl use when we specify `--kubeconfig`?

Reply:
That makes perfect sense, I was attempting to test that but I changed my development environment recently and some of my dev flow broke (I couldn't create a test cluster due to some weird certification error, etc), and I just haven't had the time to fix that yet. I can do that soon ;)